src/assets.rs

```diff
@@ -25,22 +25,23 @@ pub(super) fn plugin(app: &mut App) {
 pub struct ImageHandles(HashMap<String, Handle<Image>>);
 
 impl ImageHandles {
-    pub const KEY_DUCKY: &'static str = "Ducky";
+    pub const KEY_DUCKY: &'static str = "images/ducky.png";
 }
 
 impl FromWorld for ImageHandles {
     fn from_world(world: &mut World) -> Self {
         let asset_server = world.resource::<AssetServer>();
-        let map = [(
-            ImageHandles::KEY_DUCKY.to_string(),
-            asset_server.load_with_settings(
-                "images/ducky.png",
-                |settings: &mut ImageLoaderSettings| {
-                    settings.sampler = ImageSampler::nearest();
-                },
-            ),
-        )]
-        .into();
+
+        let pixel_art_settings = |settings: &mut ImageLoaderSettings| {
+            // Using nearest sampling to preserve pixel art style
+            settings.sampler = ImageSampler::nearest();
+        };
+        let mut map = HashMap::new();
+        map.insert(
+            Self::KEY_DUCKY.to_string(),
+            asset_server.load_with_settings(Self::KEY_DUCKY, pixel_art_settings),
+        );
```

Comment:
This could be the same `[key].into_iter().map().collect()` as the other two structs.

Reply:
[`f1b7382` (#240)](https://github.com/TheBevyFlock/bevy_quickstart/pull/240/commits/f1b73827958847f8b3759ecccefa02296362b789)